subxt/src/storage/storage_key.rs

```diff
@@ -125,53 +136,28 @@ impl<K: ?Sized> StorageKey for StaticStorageKey<K> {
             }
             .into());
         };
-        *hashers_and_ty_ids = &hashers_and_ty_ids[1..]; // Advance cursor by 1
-        decode_storage_key_from_hash(cursor, hasher, *ty_id, metadata)
-    }
-}
 
-pub fn decode_storage_key_from_hash<K: ?Sized>(
-    cursor: &mut &[u8],
-    hasher: &StorageHasher,
-    ty_id: u32,
-    metadata: &Metadata,
-) -> Result<StaticStorageKey<K>, Error> {
-    strip_storage_hash_bytes(cursor, hasher)?;
-
-    let bytes = *cursor;
-    if let Err(err) = scale_decode::visitor::decode_with_visitor(
-        cursor,
-        ty_id,
-        metadata.types(),
-        scale_decode::visitor::IgnoreVisitor,
-    ) {
-        return Err(scale_decode::Error::from(err).into());
+        // Advance the bytes cursor, returning any key bytes.
```

Comment:
With the `consume_hash_returning_bytes` function I thought it would be cleaner then to inline the other code for this impl